pkg/util/port_range.go

```diff
@@ -0,0 +1,59 @@
+package util
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type PortRange struct {
+	MinInclusive int
```

Comment:
Prefer to implement this as base + count - it's clear what is inclusive/exclusive and it's how I plan to implement port ranges for services :)